lib/travis/build/stages/skip.rb

```diff
@@ -0,0 +1,13 @@
+require 'travis/build/stages/base'
+
+module Travis
+  module Build
+    class Stages
+      class Skip < Base
+        def run
```

Comment:
We should add a friendly message that we are skipping this stage because of the special configuration; e.g.,

```ruby
sh.echo "Skipping the #{name} stage due to the configuration.", ansi: :green
```